app/uk/gov/hmrc/bindingtariffclassification/model/Pagination.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.bindingtariffclassification.model
+
+import play.api.mvc.QueryStringBindable
+
+case class Pagination
+(
+  page: Int = 1,
+  pageSize: Int = 100
```

Comment:
Defaulted this to 100 for now. The UI can pass the `page` and `page_size` query param when we play the story to add UI pagination

Reply:
This just protects the backend+frontend from falling over when theres thousands of results